src/dune_rules/command.mli

```diff
@@ -32,11 +32,14 @@ open! Import
     "../src/foo.ml" if the command is started from the "test" directory. *)
 
 module Args : sig
-  type static = Static
+  type without_targets = [ `Without_targets ]
 
-  type dynamic = Dynamic
+  type any =
+    [ `Without_targets
+    | `With_targets
+    ]
 
-  (** The type [expand] captures the meaning of a static [Command.Args.t] that
+  (** The type [expand] captures the meaning of a dynamic [Command.Args.t] that
```

Comment:
I think we can just drop the word "dynamic" here.

Reply:
Indeed, dropped